app/Resources/assets/js/lsdoc/index.js

```diff
@@ -131,5 +168,217 @@ var ImportFrameworks = (function(){
 
     return {
         fromAsn: asnStructure
+    };
+})();
+
+var Import = (function(){
+
+    var file = "";
+
+    function csvImporter(content){
+        file = content;
+
+        var lines = file.split("\n");
+        var columns = lines[0].split(",");
+        var selects = $('.select');
+
+        $.each(selects, function(){
+            var $select = $(this);
+            $select.find('option').remove().end();
+            $select.append($('<option>').val('').text(''));
+
+            columns.forEach(function(column){
+                if(column.length > 0){
+                    $select.append($('<option />').val(column).text(column));
+                }
+            });
+        });
+
+        $('#wizard').modal('hide');
+        $('#fields').modal('show');
+    }
+
+    function jsonImporter(file){
+        var json = JSON.parse(file);
+        var keys = Object.keys(json);
+        var subKey = [];
+        console.log(keys);
+
+        keys.forEach(function(subJson){
+            subKey = Object.keys(subJson);
+            console.log(subKey);
```

Comment:
We will want to make sure that we remove console.log() calls before finishing the PR as well, as some browsers do not have that method and will then throw an exception.